public/app/core/utils/query.ts

```diff
@@ -24,24 +24,27 @@ export function addQuery(queries: DataQuery[], query?: Partial<DataQuery>, datas
 export function updateQueries(
   newSettings: DataSourceInstanceSettings,
   queries: DataQuery[],
-  extensionID: string, // pass this in because importing it creates a circular dependency
+  expressionDataSourceName: string, // pass this in because importing it creates a circular dependency
```

Comment:
:thinking: with the changes in #41172 this should not be necessary:
https://github.com/grafana/grafana/pull/41172/files#diff-e66e8da9f7917f6cdf1356835e841715a074f285034ed29f6a5c7d1852fa97f7R22

Reply:
ok, reverted this name change. Looks like Mixed data source + expressions is broken in main, does   #41172  solve that?

Reply:
yes